src/elements/OcMenuItem.vue

```diff
@@ -5,10 +5,10 @@
       <slot name="default">
         <div v-if="icon" class="uk-flex uk-flex-middle">
           <oc-icon :name="icon" class="uk-margin-small-right" />
-          {{ content }}
+          <template>{{ content }}</template>
         </div>
         <template v-else>
-          {{ content }}
+          <template v-translate>{{ content }}</template>
```

Comment:
this will not work .... v-translate has to remain in phoenix.

Reply:
I removed the translations and edited slots so now it is able to use translations via the slot in Phoenix and still preserve the classes for alignment when the item has an icon.